src/AbstractAlgebra.jl

```diff
@@ -203,6 +203,12 @@ import .PrettyPrinting: expressify
 #
 ###############################################################################
 
+function check_parent(a, b, throw::Bool = true)
+   flag = parent(a) == parent(b)
```

Comment:
```suggestion
   flag = parent(a) === parent(b)
```
I saw some `===` and `==` in the old code, but is there a reason to prefer `==`?

Reply:
Happy to use `===` (I really just copied a random method and edited it, and didn't pay attention to this).